app/src/js/components/Datepicker/Datepicker.js

```diff
@@ -173,32 +163,32 @@ export class Datepicker extends React.PureComponent {
         <div className="datetime">
           <div className="datetime__range">
             <ul className="datetime__internal">
-                  <li>
-                    { this.renderDateRangeDropDown() }
-                  </li>
-                  <li>
-                    { this.renderDateTimeRange('startDateTime') }
-                  </li>
-                  <li>
-                  <span> to </span>
-                  </li>
-                  <li>
-                    { this.renderDateTimeRange('endDateTime') }
-                  </li>
-                  <li className="selector__hrformat">
-                    { this.renderHourFormatSelect() }
-                  </li>
-                </ul>
-              </div>
-            <div className="datetime__wrapper">
+              <li>
+                { this.renderDateRangeDropDown() }
+              </li>
+              <li data-cy='startDateTime'>
+                { this.renderDateTimeRange('startDateTime') }
+              </li>
+              <li>
+                <span> to </span>
+              </li>
+              <li data-cy='endDateTime'>
+                { this.renderDateTimeRange('endDateTime') }
+              </li>
+              <li className="selector__hrformat" data-cy='hourFormat'>
+                { this.renderHourFormatSelect() }
+              </li>
+            </ul>
+          </div>
+          <div className="datetime__wrapper">
```

Comment:
I know this didn't arise from this PR, but we might want to consider putting all of the `.datetime` contents into `.datetime__wrapper` and adjusting the styles accordingly so that we can avoid some of the absolute/relative positioning that's going on with this component.

Reply:
We needed the absolute/relative positioning for the calendar and clock overlays. Since we will be hiding them they will no longer be an issue. We can refactor the style but we can put that in another ticket.

Reply:
I'm calling this, future-Matt's problem.